doc/classes/GraphNode.xml

```diff
@@ -129,6 +149,16 @@
 				Returns the color set to [code]idx[/code] right (output) slot.
 			</description>
 		</method>
+		<method name="set_slot_type_left">
+			<return type="void">
+			</return>
+			<argument index="0" name="idx" type="int">
+			<argument index="1" name="type_left" type="int">
+			</argument>
+			<description>
+				Sets the (integer) type of left (input) [code]idx[/code] slot.
```

Comment:
I think that methods better borrow description from the main `set_slot` method. You can update the getters' descriptions as well. Because I don't think that that "(integer)" in parenthesis is of any use to the reader, but other information, that is currently missing here, might be.

Reply:
Done.